test/e2e/specs/all-features-confirmation/recordset/presentation.spec.ts

```diff
@@ -0,0 +1,792 @@
+      { title: 'Type of Facilities', comment: 'Type of facilities (Luxury/Upscale/Basic)' },
+      { title: 'Image Count', comment: 'Image Count' },
+      { title: 'Image Distinct Count', comment: 'Image Distinct Count' },
+      { title: 'Min Image ID', comment: 'Min Image ID' },
+      { title: 'summary of Image ID', comment: 'Summary of Image ID' },
+      { title: 'color_rgb_hex_column' }
+    ],
+    data: [
+      {
+        id: '2003',
+        title: 'NH Munich Resort',
+        website: {
+          url: 'http://www.nh-hotels.com/hotels/munich',
+          caption: 'Link to Website'
+        },
+        rating: '3.2000',
+        no_of_rooms: '15',
+        summary: 'NH Hotels has six resorts in the city of Munich. Very close to Munich Main Train Station -- the train being one of the most interesting choices of transport for travelling around Germany -- is the four-star NH München Deutscher Kaiser Hotel. In addition to the excellent quality of accommodation that it offers, the hotel is located close to Marienplatz, the monumental central square in the city, the Frauenkirche church, Stachus (Karlsplatz) and the Viktualienmarkt. Other places of interest to explore in Munich are the English garden, the spectacular Nymphenburg Palace and the German Museum, a museum of science and technology very much in keeping with the industrial spirit of the city. Do not forget to visit Munich at the end of September and beginning of October, the time for its most famous international festival: Oktoberfest! Beer, sausages, baked knuckles and other gastronomic specialities await you in a festive atmosphere on the grasslands of Theresienwiese. Not to be missed! And with NH Hotels you can choose the hotels in Munich which best suit your travel plans, with free WiFi and the possibility to bring your pets with you. NH Hotels has six resorts in the city of Munich. Very close to Munich Main Train Station -- the train being one of the most interesting choices of transport for travelling around Germany -- is the four-star NH München Deutscher Kaiser Hotel. In addition to the excellent quality of accommodation that it offers, the hotel is located close to Marienplatz, the monumental central square in the city, the Frauenkirche church, Stachus (Karlsplatz) and the Viktualienmarkt. Other places of interest to explore in Munich are the English garden, the spectacular Nymphenburg Palace and the German Museum, a museum of science and technology very much in keeping with the industrial spirit of the city. Do not forget to visit Munich at the end of September and beginning of October, the time for its most famous international festival: Oktoberfest! Beer, sausages, baked knuckles and other gastronomic specialities await you in a festive atmosphere on the grasslands of Theresienwiese. Not to be missed! And with NH Hotels you can choose the hotels in Munich which best suit your travel plans, with free WiFi and the possibility to bring your pets with you.',
```

Comment:
This summary text is the same content twice (e.g. "abc123" + "abc123"). Before this was tested using `contains` but I wanted to reuse the `testRecordsetTableRowValues`. Otherwise, the "key/value" pairs in this object were reordered to be in the same order as the columns are displayed in the recordset table.

Note: This could be improved more since the tests never use the "keys". The only value that is accessed by key is `id`.